server/src/main/java/com/defold/extender/WebSecurityConfig.java

```diff
@@ -21,16 +19,7 @@
 	@Value("${extender.authentication.platforms}")
 	String[] authenticatedPlatforms;
 
-	@Value("${extender.authentication.users}")
-	Resource usersResource;
-
-	private Properties loadUsers() throws Exception {
-		Properties users = new Properties();
-		if (usersResource != null) {
-			users.load(usersResource.getInputStream());
-		}
-		return users;
-	}
+	private InMemoryUserDetailsManager userDetailsManager = new InMemoryUserDetailsManager();
```

Comment:
We start with an empty user details manager. It will get updated before the first build via the UserUpdateService.